Vagrantfile

```diff
@@ -14,11 +14,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
  config.vm.provider :libvirt do |domain|
    domain.memory = 2048
-   domain.cpus   = 2
+   domain.cpus   = 4
```

Comment:
On 05/21/2015 09:18 AM, Chris Duryee wrote:
>> +   domain.cpus   = 4
> 
> Is this to get more |pulp_worker| instances?

Not specifically, but it does give a little more processing power to the
VM. flake8 does seem to take advantage of this, so that goes a little
faster.

-- 
Randy Barlow